packages/snjs/test/lib/004.encryption.test.ts

```diff
@@ -1,42 +1,29 @@
 import {
   Platform,
   Environment,
-  DeinitSource,
   KeyParamsOrigination,
   SNProtocolOperator004,
   ProtocolVersion
 } from '@Lib/index';
-import { createApplication } from '../setup/snjs/appFactory';
+import { createApplication } from '../factory';
 import SNCrypto from '../setup/snjs/snCrypto';
 
 describe('Test 004 encryption', () => {
-  /** The global Standard Notes application. */
-  let testSNApp;
-
   const identifier = 'hello-test@sn.org';
   const password = 'this-is-a-password-1234';
-  let rootKey;
-
   const protocol004 = new SNProtocolOperator004(new SNCrypto());
 
-  beforeEach(async () => {
-    testSNApp = await createApplication('test-application', Environment.Web, Platform.LinuxWeb);
-    rootKey = await protocol004.createRootKey(
+  it('should properly encrypt and decrypt a piece of text', async () => {
+    const application = createApplication('test-application', Environment.Web, Platform.LinuxWeb);
```

Comment:
Needs to be de-inited. We may have only one test in this file now, but more may be added in the future.